frappe/printing/page/print/print.js

```diff
@@ -709,7 +709,6 @@ frappe.ui.form.PrintView = class {
 			args: {
 				doc: this.frm.doc,
 				print_format: this.selected_format(),
-				_lang: this.lang_code,
```

Comment:
This is intentional? We pop _lang from form_dict to set language during print.

Reply:
Ah, reverted in 8f2e5563e49ebc0ffd86aa36b2eb0fd3dcf2d7e5